src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -695,7 +695,7 @@ public function getExtensionConfig($name)
             $this->extensionConfigs[$name] = [];
         }
 
-        return $this->extensionConfigs[$name];
+        return $this->parameterBag->resolveValue($this->extensionConfigs[$name]);
```

Comment:
the fix evolves around this change, but is causing different results in `testOverriddenEnvsAreMerged`. Im not sure it's expected ... cc @nicolas-grekas 

However it's needed for an out-of-the-box fix

Reply:
see https://github.com/symfony/symfony-docs/pull/11599